apps/site/assets/css/_trip-plan-results.scss

```diff
@@ -81,4 +84,24 @@
     margin-bottom: $base-spacing;
     padding: $base-spacing;
   }
+
+  &-fares {
+    min-width: 40%;
+    padding-left: 0;
+    padding-top: $base-spacing;
+
+    @include media-breakpoint-up(xsm) {
```

Comment:
I'm curious about how well this is working -- I was able to find a state where the container width is too small for adding the left padding but also too big to put the fares below the other content -- see screenshot at 396px. You might need to add some logic to enforce these to be full width in the narrowest viewports

<img width="419" alt="image" src="https://user-images.githubusercontent.com/2136286/83161593-a16de200-a0d6-11ea-8d9d-83cfa81ec1cd.png">

Reply:
Yeah, I was hesitant between 395 and 398 pixels...Are you saying that introducing `xsm` is not really a good approach? I can certainly remove this new grid breakpoint but if I do that, I'm going to need your CSS expertise, how do I detect the wrapping?

Reply:
Good questions... I'd avoid adding new breakpoints, though I could see why this new breakpoint would do well here. But probably isn't practical to try to detect _exactly_ where to transition from showing these sections side-by-side to stacked, since the left section width will vary based on itinerary. 

I think for this reason I'd remove the `flex-wrap` property... I also think we should try using the `sm` breakpoint, even though it might look spacious for many itineraries. If we add `display: block` to the smaller breakpoint it'll unset the flexboxing and let the sides stack. Hope this helps!

Reply:
I hadn't looked at it that way, thanks a lot!